### DISCUSSION TITLE
I have an error when recording 2 or more times

### DISCUSSION THREAD

**Discussion by nguyenhuyhoang241298** [katspaugh/wavesurfer.js#3251] [Q&A]
https://wavesurfer.xyz/examples/?record.js

When I record 2 or more times, I have warning like this 
![image](https://github.com/katspaugh/wavesurfer.js/assets/113531382/06c91c2f-f0e1-4900-b1c6-f99d4a15d08f)

**Comment by katspaugh** [comment]
Thanks for the report!

We'll fix it. Please use v7.3.4 in the meanwhile:

```
import RecordPlugin from 'https://unpkg.com/wavesurfer.js@7.3.4/dist/plugins/record.esm.js'
```
